### PR TITLE
New subparts for sales for the last month and for the last year

### PR DESCRIPTION
**Pull request by DennisDittmann**
Added two subparts to the dashboard: sales for the last 30 days and sales for the last 365 days

### CODE REVIEW

**Comment by aimeos** [comment]
Cool stuff, thanks a lot! :-)

Could you please replace the spaces for indention by tabs (4 spaces = 1 tab) to satisfy the code checks?
Also, can you please replace "@since 2016.07" resp. "@since 2016.04" by "@since 2018.01"?

File: admin/jqadm/templates/dashboard/item-order-saleslastmonth-default.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="order-saleslastmonth card col-lg-12">
+	<div id="order-saleslastmonth-head" class="card-header header" role="tab"
+		 data-toggle="collapse" data-target="#order-saleslastmonth-data"
+		 aria-expanded="true" aria-controls="order-saleslastmonth-data">
+		<div class="card-tools-left">
+			<div class="btn btn-card-header act-show fa"></div>
+		</div>
+		<span class="item-label header-label">
+			Umsatzentwicklung der letzten 30 Tage
```
Comment (aimeos) [inline admin/jqadm/templates/dashboard/item-order-saleslastmonth-default.php]: Can you please translate this to English and add
```<?= $enc->html( $this->translate( 'admin', '...' ) ); ?>```
around?

File: admin/jqadm/templates/dashboard/item-order-saleslastyear-default.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="order-saleslastyear card col-lg-12">
+	<div id="order-saleslastyear-head" class="card-header header" role="tab"
+		 data-toggle="collapse" data-target="#order-saleslastyear-data"
+		 aria-expanded="true" aria-controls="order-saleslastyear-data">
+		<div class="card-tools-left">
+			<div class="btn btn-card-header act-show fa"></div>
+		</div>
+		<span class="item-label header-label">
+			Gesamtumsatz der letzten 12 Monate
```
Comment (aimeos) [inline admin/jqadm/templates/dashboard/item-order-saleslastyear-default.php]: Can you please translate this to English and add
```<?= $enc->html( $this->translate( 'admin', '...' ) ); ?>```
around?

File: admin/jqadm/themes/dashboard.js [hunk 3]
```diff
@@ -689,6 +697,219 @@ Aimeos.Dashboard.Order = {
...
+			.attr("transform", "translate(" + margin.left + "," + margin.top + ")");
+
+		svg.append("g")
+			.attr("class", "x axis")
+			.attr("transform", "translate(0," + height + ")")
+			.call(xAxis);
+
+		var criteria = {"&&": [
+			{">=": {"order.cdate": firstdate.toISOString().substr(0, 10)}},
+			{"||": [
```
Comment (aimeos) [inline admin/jqadm/themes/dashboard.js]: You can shorten this to `{">=": {"order.statuspayment": 5}}`

File: admin/jqadm/themes/dashboard.js [hunk 4]
```diff
@@ -689,6 +697,219 @@ Aimeos.Dashboard.Order = {
...
```
Comment (aimeos) [inline admin/jqadm/themes/dashboard.js]: Same here: `{">=": {"order.statuspayment": 5}}`

File: admin/jqadm/themes/dashboard.js [hunk 5]
```diff
@@ -689,6 +697,219 @@ Aimeos.Dashboard.Order = {
...
+				.data(data.data).enter()
+				.append("g").attr("class", "barsum");
+
+			bars.append("rect")
+				.attr("class", "bar sum")
+				.attr("x", function(d) { return (xScale((d.id.substr(5,2))) + xScale.bandwidth() / 2 + 1); })
+				.attr("width", xScale.bandwidth() / 2 - 1)
+				.attr("y", function(d) { return yScale(+d.attributes); })
+				.attr("height", function(d) { return height - yScale(+d.attributes); })
+				.append("title").text(function(d) { return d.attributes.replace('.', ',') + ' €'; });
```
Comment (aimeos) [inline admin/jqadm/themes/dashboard.js]: You should use the `numFmt` variable to format the number according to the browsers locale. What if the currency isn't `€`?

File: admin/jqadm/themes/dashboard.js [hunk 6]
```diff
@@ -88,16 +88,16 @@ Aimeos.Dashboard = {
 
 			var svg = d3.select(selector)
 				.append("svg")
-					.attr("width", width + margin.left + margin.right)
-					.attr("height", (width - lgspace > height ? height + margin.top + margin.bottom : radius * 2 + margin.top + data.data.length * txheight + 25 ));
+				.attr("width", width + margin.left + margin.right)
```
Comment (aimeos) [inline admin/jqadm/themes/dashboard.js]: Ups, you removed the indentions of the other code by accident now ...

Comment (DennisDittmann) [reply admin/jqadm/themes/dashboard.js]: Yes, stupid :) Changed this.

**Comment by aimeos** [comment]
Great! We will care about the rest
